src/Markdig.Wpf/Renderers/WpfRenderer.cs

```diff
@@ -10,49 +10,52 @@
 using System.Runtime.CompilerServices;
 using System;
 using Markdig.Annotations;
+using System.Linq;
+using Markdig.Wpf;
+using System.Windows.Markup;
+using Markdig.Renderers.Wpf.Extensions;
 
 namespace Markdig.Renderers
 {
     /// <summary>
     /// WPF renderer for a Markdown <see cref="Syntax.MarkdownDocument"/> object.
     /// </summary>
-    /// <seealso cref="Renderers.TextRendererBase{T}" />
+    /// <seealso cref="Renderers.RendererBase" />
     public class WpfRenderer : RendererBase
     {
-        private readonly Stack<Block> blocks = new Stack<Block>();
-        private readonly List<Inline> inlines = new List<Inline>();
+        private readonly Stack<IAddChild> stack = new Stack<IAddChild>();
         private char[] buffer;
 
         public WpfRenderer(FlowDocument document)
         {
             buffer = new char[1024];
             Document = document;
+            document.SetResourceReference(Paragraph.StyleProperty, Styles.DocumentStyleKey);
+            stack.Push(document);
 
             // Default block renderers
-            //ObjectRenderers.Add(new CodeBlockRenderer());
-            //ObjectRenderers.Add(new ListRenderer());
-            //ObjectRenderers.Add(new HeadingRenderer());
-            //ObjectRenderers.Add(new HtmlBlockRenderer());
+            ObjectRenderers.Add(new CodeBlockRenderer());
+            ObjectRenderers.Add(new ListRenderer());
+            ObjectRenderers.Add(new HeadingRenderer());
             ObjectRenderers.Add(new ParagraphRenderer());
-            //ObjectRenderers.Add(new QuoteBlockRenderer());
-            //ObjectRenderers.Add(new ThematicBreakRenderer());
+            ObjectRenderers.Add(new QuoteBlockRenderer());
+            ObjectRenderers.Add(new ThematicBreakRenderer());
 
             // Default inline renderers
-            //ObjectRenderers.Add(new AutolinkInlineRenderer());
-            //ObjectRenderers.Add(new CodeInlineRenderer());
+            ObjectRenderers.Add(new AutolinkInlineRenderer());
+            ObjectRenderers.Add(new CodeInlineRenderer());
             ObjectRenderers.Add(new DelimiterInlineRenderer());
-            //ObjectRenderers.Add(new EmphasisInlineRenderer());
+            ObjectRenderers.Add(new EmphasisInlineRenderer());
             ObjectRenderers.Add(new LineBreakInlineRenderer());
-            //ObjectRenderers.Add(new HtmlInlineRenderer());
-            //ObjectRenderers.Add(new HtmlEntityInlineRenderer());
-            //ObjectRenderers.Add(new LinkInlineRenderer());
+            ObjectRenderers.Add(new LinkInlineRenderer());
             ObjectRenderers.Add(new LiteralInlineRenderer());
+
+            // Extension renderers
+            ObjectRenderers.Add(new TaskListRenderer());
```

Comment:
Again, this will need to be reviewed regarding the use of extensions. Ideally renderers should only be added if the extension is active.